ci/build_container/docker_push.sh

```diff
@@ -27,7 +27,10 @@ then
         cd ci/build_container
         docker login -u "$DOCKERHUB_USERNAME" -p "$DOCKERHUB_PASSWORD"
 
-        for distro in ubuntu centos
+        # TODO(mattklein123): centos has been removed because the version of cmake is too old to
```

Comment:
Optional: arguably shouldn't be your TODO if you're not going to be fixing it it :-P